public/app/core/components/AppChrome/NewsContainer.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+
+import { Drawer, Icon } from '@grafana/ui';
+import { DEFAULT_FEED_URL } from 'app/plugins/panel/news/constants';
+
+import { NewsWrapper } from './NewsWrapper';
+
+interface NewsContainerProps {
+  buttonCss?: string;
+}
+
+export function NewsContainer({ buttonCss }: NewsContainerProps) {
+  const [showNewsDrawer, setShowNewsDrawer] = useState(false);
+
+  const onChildClick = () => {
+    setShowNewsDrawer(true);
+  };
```

Comment:
the useToggle hook is kind of nice in situtations like this as you don't need the callback, and 
onClose={() => setShowNewsDrawer(false)}>
become onClose={onToggleShowNewsDrawer}>